apps/lockfile-explorer-web/src/containers/LockfileViewer/index.tsx

```diff
@@ -155,11 +165,16 @@ export const LockfileViewer = (): JSX.Element | ReactNull => {
           }
         ]}
       />
-      <div className={`${appStyles.ContainerCard} ${styles.ViewWrapper}`}>
+      <div className={`${styles.ViewContents} ${appStyles.ContainerCard}`}>
         <div className={styles.LockfileFilterBar}>
-          {entryStack.length > 1 ? <button onClick={pop}>back</button> : null}
-          <h5>filter:</h5>
+          <h5>Filter:</h5>
```

Comment:
Yeah I think that makes sense for the Filter to be applicable to each tab individually, so I changed it so that it saves and uses different filter text depending on which tab your in, so they can search for things independently.

As for the checkboxes, I feel that it would make sense that they affect both tabs as the user would probably want to look for the same "hits" across both tabs, but to make it a bit more clear that it's a separate entity I've added a border to separate the list and the filter checkbox area, let me know what you think:

![image](https://user-images.githubusercontent.com/8737494/202531019-e32885e0-1fd0-46c0-ae64-c6c20a9b52d5.png)